lib/model/query/entities.js

```diff
@@ -94,13 +94,14 @@ const createMany = (dataset, entities, sourceId, userAgentIn) => async ({ all, c
   const creatorId = context.auth.actor.map((actor) => actor.id).orNull();
   const userAgent = blankStringToNull(userAgentIn);
 
-  const entityInsert = sql.join(entities.map(e => sql`(${sql.join([dataset.id, e.uuid, creatorId, sql`clock_timestamp()`], sql`,`)} )`), sql`,`);
+  const entityRows = entities.map(e => [dataset.id, e.uuid, creatorId]);
+  const entityColumnTypes = ['int4', 'uuid', 'int4'];
```

Comment:
Postgres does have a `uuid` data type, but I don't think we store `entities.uuid` that way. It looks like the column is `character varying(255)`.